packages/client/components/InsightsDomainNudge.tsx

```diff
@@ -0,0 +1,113 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import useModal from '../hooks/useModal'
+import CreditCardModal from '../modules/userDashboard/components/CreditCardModal/CreditCardModal'
+import {PALETTE} from '../styles/paletteV3'
+import {InsightsDomainNudge_domain$key} from '../__generated__/InsightsDomainNudge_domain.graphql'
+import PrimaryButton from './PrimaryButton'
+
+const NudgeBlock = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  borderTop: `1px solid ${PALETTE.SLATE_400}`,
+  padding: 16,
+  width: '100%'
+})
+
+const OverLimitBlock = styled('div')({
+  backgroundColor: PALETTE.GOLD_100,
+  borderRadius: 2,
+  color: PALETTE.SLATE_900,
+  fontSize: 16,
+  lineHeight: '24px',
+  width: '100%'
+})
+
+const OverLimitCopy = styled('div')({
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  padding: '4px 12px'
+})
+
+const CTA = styled(PrimaryButton)({
+  lineHeight: '24px',
+  padding: '8px 32px',
+  width: 'fit-content'
+})
+
+const ButtonBlock = styled('div')({
+  paddingTop: 16
+})
+
+interface Props {
+  domainRef: InsightsDomainNudge_domain$key
+}
+
+const InsightsDomainNudge = (props: Props) => {
+  const {domainRef} = props
+  const domain = useFragment(
+    graphql`
+      fragment InsightsDomainNudge_domain on Company {
+        id
+        suggestedTier
+        tier
+        organizations {
+          id
+          name
+          tier
+          orgUserCount {
+            activeUserCount
+          }
+        }
+      }
+    `,
+    domainRef
+  )
+  const {id: domainId, suggestedTier, tier, organizations} = domain
+  const personalOrganizations = organizations
+    .filter((org) => org.tier === 'personal')
+    .sort((a, b) => (a.orgUserCount > b.orgUserCount ? -1 : 1))
+  const [biggestOrganization] = personalOrganizations
+  const organizationName = biggestOrganization?.name ?? ''
+  const suggestPro = suggestedTier === 'pro' && tier === 'personal'
+  const suggestEnterprise = suggestedTier === 'enterprise' && tier !== 'enterprise'
+  const showNudge = suggestPro || suggestEnterprise
+  const CTACopy = suggestPro ? `Upgrade ${organizationName} to Pro` : 'Contact Us'
+  const onClickCTA = () => {
+    if (suggestPro) {
+      togglePortal()
+    } else if (suggestEnterprise) {
+      window.open('mailto:love@parabol.co?subject=Increase Usage Limits')
+    }
+  }
+  const {togglePortal, closePortal, modalPortal} = useModal()
+  if (!showNudge) return null
+  return (
+    <NudgeBlock>
+      <OverLimitBlock>
+        <OverLimitCopy>
+          <b>{domainId}</b> is over the limit of <b>2 Free Teams</b>
+        </OverLimitCopy>
+      </OverLimitBlock>
+      <ButtonBlock>
+        <CTA size={'large'} onClick={onClickCTA}>
+          {CTACopy}
+        </CTA>
+      </ButtonBlock>
+      {biggestOrganization &&
+        modalPortal(
+          <CreditCardModal
```

Comment:
+1 after upgrading to pro, I don't get any confetti 🥲 

-1 after upgrading to pro, when the modal closes, the CTA button remains the same until I refresh the page. Some users might think that the upgrade wasn't successful.

Reply:
> someone with a verified email with that matching domain

Should users with verified email addresses be able to see data for other orgs within the same company? I wonder whether BigCo would be happy with BigCo Europe Org having access to BigCo North America Org's data. 

+2 great stuff fixing the confetti 🎉  

I think the PR is good to go, but the Upgrade to Pro CTA doesn't update after upgrading. I think the button should disappear immediately after so the user knows the upgrade was successful.

Reply:
> Should users with verified email addresses be able to see data for other orgs within the same company? I wonder whether BigCo would be happy with BigCo Europe Org having access to BigCo North America Org's data.

it's a really good question, i think it's the only way! if i tell you your company is using 100 seats & you only think you're using 10, you're gonna ask me who else is using parabol & i'll show ya all the verified user email addresses.

 > I think the PR is good to go, but the Upgrade to Pro CTA doesn't update after upgrading. I think the button should disappear immediately after so the user knows the upgrade was successful.

it goes away for me! i think this is the bug that only happens for superusers